app/jobs/health/process_enrollment_changes_job.rb

```diff
@@ -9,6 +9,13 @@ class ProcessEnrollmentChangesJob < ApplicationJob
     def perform(enrollment_id)
       enrollment = Health::Enrollment.find(enrollment_id)
 
+      pidsls = Health::Cp.all.map { |cp| cp.pid + cp.sl }
+      receiver_id = enrollment.receiver_id
+      unless pidsls.include?(receiver_id)
```

Comment:
Do we need to limit this to only enrollments?  (As in a dis-enrollment could be ok?)

Reply:
`enrollment` is an 834 enrollment file. Would renaming the variable help?